src/lib/operators/join_sort_merge/radix_cluster_sort.hpp

```diff
@@ -325,20 +330,25 @@ class RadixClusterSort {
   RadixClusterOutput<T> execute() {
     RadixClusterOutput<T> output;
 
+    Timer timer_materialization;
     // Sort the chunks of the input tables in the non-equi cases
     ColumnMaterializer<T> left_column_materializer(!_equi_case, _materialize_null_left);
     ColumnMaterializer<T> right_column_materializer(!_equi_case, _materialize_null_right);
     auto [materialized_left_segments, null_rows_left, samples_left] =
         left_column_materializer.materialize(_left_input_table, _left_column_id);
+    _performance.set_step_runtime(JoinSortMerge::OperatorSteps::LeftSideMaterializing, timer_materialization.lap());
     auto [materialized_right_segments, null_rows_right, samples_right] =
         right_column_materializer.materialize(_right_input_table, _right_column_id);
+    _performance.set_step_runtime(JoinSortMerge::OperatorSteps::RightSideMaterializing, timer_materialization.lap());
     output.null_rows_left = std::move(null_rows_left);
```

Comment:
Can you pull that null std::move directly behind the materialize side? So L343 after l338. Then we include the null move into the materialization side.